addons/im_livechat/static/src/core/common/transcript_sender.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.TranscriptSender">
+        <div class="o-livechat-TranscriptSender input-group has-validation">
```

Comment:
Style should match create lead/ticket panels. It might be early to create a generic component, but we should have least have the same visual.